20_classes_lab/01.py

```diff
@@ -4,3 +4,23 @@
 Complete the class so correct result is printed
 """
 
+class Summer(object):
+    def __init__(self):
+        self.sum = 0        
+
+    def add(self, *values):
+        for value in values:
```

Comment:
Better to use python's sum: `self.sum += sum(values)`. Also `sum` is a confusing name for member field (because of the builtin function)